com.lachhebo.Gabtag.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "com.lachhebo.Gabtag",
```

Comment:
this needs to be a real domain name that you control.

Reply:
e.g. in this case com.github.lachebo.Gabtag works.

Reply:
Yeah, we expect the domain to be owned by developer, or Github/GitLab/etc domain instead.

Reply:
Okay, do you think i should rename the app git.lachhebo.Gabtag ?

Reply:
` com.github.lachebo.Gabtag`, apart from `app-id`, the file itself needs to be renamed.

Reply:
Okay